### PR TITLE
Apply "Compute" branding change.

### PR DESCRIPTION
**Pull request by harmony7**

### CODE REVIEW

File: src/server/serve-static.ts [hunk 1]
```diff
@@ -11,7 +11,7 @@ import { ComputeJsNextRequest, ComputeJsNextResponse } from "./base-http/compute
 
 /**
  * Serves the contents of a file at a path.
- * (A reimplementation for Compute@Edge of function in Next.js of the same name,
+ * (A reimplementation for Compute of function in Next.js of the same name,
```
Comment (kailan) [inline src/server/serve-static.ts:14]: Is this "of" meant to be here?

File: src/util/etag.ts [hunk 2]
```diff
@@ -10,7 +10,7 @@ import crypto from 'crypto';
 
 /**
  * Generates an etag string based on a payload.
- * (An adaptation for Compute@Edge of function in Next.js of the same name,
+ * (An adaptation for Compute of function in Next.js of the same name,
```
Comment (kailan) [inline src/util/etag.ts:13]: Same here

File: src/server/serve-static.ts (hunk 1 above)
Comment (harmony7) [reply src/server/serve-static.ts:14]: yes, this is correct.  a function with this name exists in Next.js, and this is a reimplementation for Compute of that function.

File: src/util/etag.ts (hunk 2 above)
Comment (harmony7) [reply src/util/etag.ts:13]: same

**Review by kpfleming — APPROVED** [review]

**Pull request merged**